src/main/java/net/datafaker/internal/helper/WordUtils.java

```diff
@@ -1,13 +1,13 @@
 package net.datafaker.internal.helper;
 
-import java.util.Locale;
 
 public class WordUtils {
 
     public static String capitalize(String input) {
         if (input == null) return null;
-        if (input.equals("")) return "";
-        if (Character.isUpperCase(input.charAt(0))) return input;
-        return input.substring(0, 1).toUpperCase(Locale.ROOT) + input.substring(1);
+        if (input.isEmpty()) return "";
```

Comment:
Why not just return input, instead of instantiating a litteral?

Reply:
good point, sorry didn't notice and merged